src/test/resources/json/updateexportlocation_mutation_all.json

```diff
@@ -0,0 +1,9 @@
+{
+  "query": "mutation updateExportLocation($updateExportLocationInput:UpdateExportLocationInput!){updateExportLocation(exportLocation:$updateExportLocationInput)}",
+  "variables": {
+    "updateExportLocationInput": {
+      "consignmentId": "6e3b76c4-1745-4467-8ac5-b4dd736e1b3e",
+      "exportLocation": "6e3b76c4-1745-4467-8ac5-b4dd736e1b3e.tar.gz"
```

Comment:
This has made me think: do you know whether the export location will be a full S3 URL, or a path within a bucket? I don't really mind either way - I guess it depends on how it will be used. If it's an S3 URL, should we validate the structure?

Reply:
The way I've written the consignment export at the moment it's a path within the bucket. I think it's fine for now. We know which bucket it's going to be in. If it turns out to be a problem later, we can always change it.